### PR TITLE
added another submit button on the form page so that the redirect goe…

### PR DESCRIPTION
**Pull request by madoleary**
* [X ] Are you working on the latest release?
* [X] Have you tested it locally?
* [FEATURE ] Please mention if it is a fix, enhancement or feature
* [ X] Please explain your change -> added another submit button on the form page so that the redirect goes back to the form page in case the user wants to add another point (thinking about a more elegant ajax solution)

Thanks !

### CODE REVIEW

**Comment by piks3l** [comment]
Waaah that looks amazing. I'll try it out right now !

**Comment by piks3l** [comment]
Works for me ! Amazing ! Thank you so much @madoleary, going into production right now !

File: db/migrate/20171214143221_create_comments.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class CreateComments < ActiveRecord::Migration[5.1]
   def change
+    drop_table :comments
```
Comment (piks3l) [inline db/migrate/20171214143221_create_comments.rb]: Hey @madoleary I forgot to ask, why did you put that there ?

Comment (madoleary) [reply db/migrate/20171214143221_create_comments.rb]: db:migrate wasn't working for me with an error occurring with respect to comments so i dropped the table before completing the migration.

Comment (piks3l) [reply db/migrate/20171214143221_create_comments.rb]: Ok, are you ok if I comment that out in production ? If it doesn't work, well... we'll fix it ahaha

Comment (madoleary) [reply db/migrate/20171214143221_create_comments.rb]: sure, should have commented it out before pushing, so my bad.

Comment (piks3l) [reply db/migrate/20171214143221_create_comments.rb]: okidokie

**Pull request merged**